src/component/term/TermMetadataEdit.tsx

```diff
@@ -205,6 +214,19 @@ export class TermMetadataEdit extends React.Component<TermMetadataEditProps, Ter
                                              help={i18n("term.source.help")}/>
                             </Col>
                         </Row>
+                        <Row>
+                            <Col xs={12}>
+                                <FormGroup>
+                                    <Label id="term-metadata-edit-status" className="attribute-label">
+                                        {i18n("term.metadata.status")}
+                                    </Label>
+                                    <br/>
+                                    <DraftToggle draft={this.state.draft === undefined || this.state.draft}
```

Comment:
This logic (draft === undefined) implies draft true is replicated on several places. Consider extracting it into some method.

Reply:
This belongs to #45 .